test/SdkTests/SdkTests.csproj

```diff
@@ -254,11 +242,6 @@
     <SwrFile Remove="@(SwrFile)" />
   </ItemGroup>
 
-  <PropertyGroup>
-    <TargetPath></TargetPath>
-    <!-- Prevent projects referencing this from trying to pass us to the compiler -->
-  </PropertyGroup>
```

Comment:
This makes the project inoperable in Visual Studio as the project system requires that TargetPath have a value.

Reply:
Should that be conditioned on the `IsBuildingInVisualStudio` property or something?

Reply:
> Should that be conditioned on the IsBuildingInVisualStudio property or something?

I'll respond to your question with a question, does `<TargetPath></TargetPath>` have a purpose? The comment under it implies that `SdkTests.csproj` is referenced by other projects, but it isn't. Nothing references this project.